igbominadeveloper/questioner
# 163606210 pull in changes into assets and views

#### What does this PR do?
merges changes in gh-pages into develop

#### Description of Task to be completed?
>  merges changes in gh-pages into develop

#### How should this be manually tested?
- run ```git clone https://github.com/igbominadeveloper/questioner```
- checkout the ```ch-merge-ui-changes-163606210``` branch
- check project ```ui``` folder
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?(if applicable)
https://www.pivotaltracker.com/story/show/163606210
